cp/cp_config.c

```diff
@@ -353,8 +353,8 @@ fetch_s1u_sgw_ip(uint32_t dpId)
 		}
 	}
 
-	rte_panic("Could not find s1u ip address for dpid: %u\n", dpId);
-	rte_exit(EXIT_FAILURE, "Could not find s1u ip address for dpid: %u\n", dpId);
+	rte_panic("Could not find s1u ip address for an unregistered DP\n");
```

Comment:
this function is called in the steady state of CP. one DP should not crash the CP. can we convert the panic to log and return error?

Reply:
Will create a new PR for this. Merging this current PR with the current changes.